build.sbt

```diff
@@ -45,6 +45,7 @@ libraryDependencies ++= {
     ("com.google.apis" % "google-api-services-plus" % "v1-rev381-1.20.0"),
     ("com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev53-1.20.0"),
     "com.google.apis" % "google-api-services-oauth2" % "v1-rev112-1.20.0",
+    "com.google.apis" % "google-api-services-genomics" % "v1-rev88-1.22.0",
```

Comment:
I would update this to `v1-rev89-1.22.0` according to the latest available version here:

https://oss.sonatype.org/#nexus-search;quick~google-api-services-genomics